Sources/SpeziOnboarding/OnboardingDataSource.swift

```diff
@@ -43,9 +43,9 @@ public class OnboardingDataSource: Module, EnvironmentAccessible {
     /// Adds a new exported consent form represented as `PDFDocument` to the ``OnboardingDataSource``.
     ///
     /// - Parameter consent: The exported consent form represented as `PDFDocument` that should be added.
-    public func store(_ consent: PDFDocument) async {
+    public func store(_ consent: PDFDocument, identifier: String) async {
         Task { @MainActor in
```

Comment:
I am not quite sure why we dispatch to an other task here; I think we should aim to not create too many tasks between different calls and would therefore suggest to remove this here.

Reply:
I agree, a separate task should not be necessary. I have removed it.